### PR TITLE
Update README and linting

### PR DESCRIPTION
**Pull request by izilla**
The README is not exactly accurate for the the main options

Updates the README to be more accurate and some linting to clean
things up a bit :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,10 @@
 
 ## Demo
 
-http://lightninglu10.react-slide-menu.hellodeploy.com/
```
Comment (izilla) [inline README.md]: This link goes nowhere and 404s 🤷‍♂

File: README.md [hunk 2]
```diff
@@ -11,11 +11,10 @@
...
-
-![Alt Text](https://github.com/quantfive/react-slide-menu/blob/master/slide.gif?raw=true)
+![Slide menu demo](https://github.com/quantfive/react-slide-menu/blob/master/slide.gif?raw=true)
```
Comment (izilla) [inline README.md]: Some better alt text here

File: README.md [hunk 3]
```diff
@@ -51,36 +49,43 @@ var nav = [
...
 ]
+
 <SlideMenu
   active={this.state.slideMenuActive}
   nav={nav}
   reactRouter={true}
   closeMenu={() => this.setState({slideMenuActive: false})}>
   <YOUR APP />
-</SlideMenu
+</SlideMenu>
```
Comment (izilla) [inline README.md]: Missing closing angle bracket 😨

File: README.md [hunk 4]
```diff
@@ -51,36 +49,43 @@ var nav = [
...
 ]
+
 <SlideMenu
   active={this.state.slideMenuActive}
   nav={nav}
   reactRouter={true}
   closeMenu={() => this.setState({slideMenuActive: false})}>
-  <YOUR APP />
-</SlideMenu
+  <YOUR_APP />
```
Comment (izilla) [inline README.md]: This just made the highlighting in my editor more consistent, maybe `YourApp` would be a better choice 🤷‍♂

File: README.md [hunk 5]
```diff
@@ -51,36 +49,43 @@ var nav = [
...
+| navLinkStyle     | String | Extra prop to style the `<a>` or `Link` element of each Navigation Item. Default: `linkStyle: { textDecoration: 'none', color: '#fff', fontWeight: '300', '@media only screen and (min-width: 768px)': { fontSize: '2em', }, '@media only screen and (min-width: 1440px)': { fontSize: '2.5em', }},` |
+| linkClassName    | String | Extra class to style the `<a>` or `Link` element of each Navigation item. Has the same style default as above `navLinkStyle`.                                                                                                                                                                        |
+| navClassName     | String | Extra class to style the whole Nav component.                                                                                                                                                                                                                                                        |
```
Comment (izilla) [inline README.md]: Looks a lot better with word wrap turned off 💅

File: README.md [hunk 6]
```diff
@@ -51,36 +49,43 @@ var nav = [
...
+| :------------------- | :-------- | :---------------------------------------------------------------------------------------------------------------------------------- |
+| active               | Boolean   | `true` will show the menu and `false` will hide the menu. (Required)                                                                |
+| nav                  | Array     | An array of objects that describe the navigation menu. Each object needs to have keys `id`, `label`, and `path`. (Required)         |
+| reactRouter          | Boolean   | `true` if you use React Router in your project `false` if you don't.                                                                |
+| extraComponentTop    | Component | Displayed before the Nav you can add anything you want, add a component for a logo or something else like so: `<img src={LOGO} />`. |
+| extraComponentBottom | Component | Displayed after the Nav you can add anything you want, add a component for a logo or something else like so: `<img src={LOGO} />`.  |
```
Comment (izilla) [inline README.md]: These two are not mentioned correctly but were previously `extraComponent` which does nothing there are `extraComponenetTop` and `extraComponentBottom` which are real

**Review by lightninglu10 — APPROVED** [review]
Merging

**Comment by lightninglu10** [comment]
Thanks for the README update @izilla

**Pull request merged**